Applications/Games/Overwatch/Online/script.json

```diff
@@ -5,7 +5,7 @@
         "LINUX"
     ],
     "testingOperatingSystems" : [
-        "LINUX"
+        "MACOSX"
```

Comment:
Please add MACOSX to the previous field too

Reply:
Fixed